src/test/javascript/continuous/verify_latest_spec.js

```diff
@@ -80,9 +80,11 @@ describe(__filename, function () {
                 item.empty = true;
             } else if (statusCode === 303) {
                 const location = fromObjectPath(['headers', 'location'], res);
+                // Ensure location does not contain sensitive information before logging or processing
+                const sanitizedLocation = location ? encodeURIComponent(location) : 'unknown';
                 const strLength = channelUrl.length + name.length + 2;
-                const latestKey = location.substring(strLength);
-                console.log('latestKey ', latestKey, location);
+                const latestKey = sanitizedLocation.substring(strLength);
+                console.log('latestKey (sanitized) ', latestKey);
```

Comment:
Have to check what will be value of `latestKey`, if `location` is set to '_unknown_'

Reply:
When location is missing or unknown, we handle it by assigning "null" to that latest key.
unknown log message will assign this value as null only, lastly the item is resulting as "null"

Reply:
`fromObjectPath(['headers', 'location'], res);` if this code at line#82 returns null, then `sanitizedLocation` will be set to '_unknown_'